util.py

```diff
@@ -471,6 +480,28 @@ def is_parameters_ordered_and_semantically_correct(since, name, params, protocol
     return is_ordered and is_semantically_correct
 
 
+def contains_invalid_nullability_field(params):
+    for param in params:
+        # According to the readme, UUID fields should be nullable. But a lot of codecs violate that now,
+        # so skipping that check
+        """ 
+        if param["type"] == "UUID" and not param["nullable"]:
+            print(
+                'Check the nullable value of "%s" field of the "%s".\n'
+                "UUID fields should be nullable!"
+                % (param["name"], param["since"])
```

Comment:
Instead of `since`, you need the message name here?

Reply:
see https://github.com/hazelcast/hazelcast-client-protocol/pull/494/commits/80b844a7cd7e347f66a4b8d0b5fb0671f021ab64